### ISSUE TITLE
Should be able to disable dimensions

### ISSUE THREAD

**Issue by owlas** [lightdash/dbt2looker#21]
Sometimes you have a column just needed for measures and the auto-created dimension can confuse things in the view

**Comment by owlas** [comment]
Two suggestions:

**Allow disabling dimensions**

```yaml
columns:
 - name: mycol
   meta:
      dimension:
         enabled: false
```

to disable or enable dimensions.

**Auto disable columns with measures?**

**Comment by owlas** [comment]
Closed in v0.8.1
